morpheus/_lib/tests/test_morpheus.cpp

```diff
@@ -17,19 +17,51 @@
 
 #include "test_morpheus.hpp"
 
+#include "morpheus/io/data_loader_registry.hpp"
+#include "morpheus/io/loaders/all.hpp"
+#include "morpheus/messages/meta.hpp"
+
+#include <pybind11/pybind11.h>
+
+#include "test_morpheus.hpp"
+
 #include <pybind11/embed.h>
 
 #include <cstdlib>
 #include <filesystem>
+#include <memory>
+#include <random>
+#include <sstream>
 #include <stdexcept>
 
+namespace {
+std::string accum_merge(std::string lhs, std::string rhs)
+{
+    if (lhs.empty())
+    {
+        return std::move(rhs);
+    }
+
+    return std::move(lhs) + "," + std::move(rhs);
+}
```

Comment:
We have a similar function in `StringUtil`. Can that be used?

Reply:
Yeah, that function works.